src/content/en/updates/2020/05/devtools.md

```diff
@@ -171,6 +171,20 @@ References:
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/5f6928b9570ed35c4a0b4c1de9b0d1f62370c636 #}
 
+
+## App shortcuts support in the Manifest pane
+
+App shortcuts help users quickly start common or recommended tasks within a web
+app. The app shortcuts menu is shown only for Progressive Web Apps that are
+installed on the user's desktop or mobile device.
+
+Check out [Get things done quickly with app
+shortcuts](https://web.dev/app-shortcuts) to learn more.
+
+![App shortcuts in the Manifest pane.](/web/updates/images/2020/05/app-shortcuts.png)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/9fee874360dbc0f891609a8aa1a9f8947b510253 #}
```

Comment:
@kaycebasques I've added this line to be consistent with others but I'm not sure about it is used.

Reply:
LGTM